plugins/catalog/src/components/SystemDiagramCard/SystemDiagramCard.tsx

```diff
@@ -262,6 +262,8 @@ export function SystemDiagramCard() {
         nodeMargin={10}
         direction={DependencyGraphTypes.Direction.BOTTOM_TOP}
         renderNode={RenderNode}
+        paddingX={50}
```

Comment:
Seems slightly preferable to use theme spacing here, like:
```
const theme = useTheme();
...
return (
  <InfoCard title="System Diagram">
    <DependencyGraph
      paddingX={theme.spacing(4)}
      paddingY={theme.spacing(4)}
 ```

Reply:
Verified that it works well with theme spacing, updated in code.